source/application-platform.html.md.erb

```diff
@@ -271,8 +271,6 @@ Also update the Slack alert and release channels in the Circle build (.circleci/
 
 ### Configuring the build pipeline
 
-We still have some configuration to do. If you go to the CircleCI project you should see that the build failed. If you run command `kubectl -n <your-new-service-name>-dev get pods` there are errors with the pods. The deployment failed due to missing secrets. Secrets are the kind of thing you can't save in a template repository.
```

Comment:
The Kotlin template should deploy to dev first time thanks to @mattops https://github.com/ministryofjustice/dps-project-bootstrap/pull/106